<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VmSettings_ui</class>
 <widget class="QDialog" name="VmSettings_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Virtual Machine Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="generalButton">
     <property name="text">
      <string>General</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="isoButton">
     <property name="text">
      <string>ISO</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="hardwareButton">
     <property name="text">
      <string>Hardware</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="userButton">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="proxiesButton">
     <property name="text">
      <string>Proxies</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>VM Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leComment">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leVmName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkVmware">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>VMware</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkVbox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Virtualbox</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkParallels">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Paralells</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkDesktop">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Desktop</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkUpdate">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>100</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Updates</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkVagrant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>86</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vagrant</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Iso Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Iso URL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Iso Hash</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ISO Hash Algorithm</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leIsoName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leIsoUrl">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leIsoHash">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leIsoPath">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Iso Path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leIsoHashAlgorithm">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CPUs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Memory Size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Disk Size</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leCpus">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leMemSize">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leDiskSize">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLineEdit" name="leUserName">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserPassword">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="leVerifyPassword">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserHomeDir">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserShell">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="laUserName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ssh User Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="laUserPassword">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ssh User Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="laVerifyPassword">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Verify Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="laUserHomeDir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>User Home Dir</string>
       </property>
      </widget>
      <widget class="QLabel" name="laUserShell">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>User Shell</string>
       </property>
      </widget>
      <widget class="QLabel" name="laUserComment">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>User Long Name/Comment</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserComment">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QLabel" name="laHttp_proxy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>http_proxy</string>
       </property>
      </widget>
      <widget class="QLabel" name="laHttps_proxy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>https_proxy</string>
       </property>
      </widget>
      <widget class="QLabel" name="laFtpProxy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ftp_proxy</string>
       </property>
      </widget>
      <widget class="QLabel" name="laRsyncProxy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>rsync_proxy</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leHttpProxy">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leHttpsProxy">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leFtpProxy">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRsyncProxy">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="laNoProxy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>no_proxy</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNoProxy">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>generalButton</tabstop>
  <tabstop>isoButton</tabstop>
  <tabstop>hardwareButton</tabstop>
  <tabstop>userButton</tabstop>
  <tabstop>proxiesButton</tabstop>
  <tabstop>leComment</tabstop>
  <tabstop>leVmName</tabstop>
  <tabstop>leIsoName</tabstop>
  <tabstop>leIsoUrl</tabstop>
  <tabstop>leIsoHash</tabstop>
  <tabstop>leCpus</tabstop>
  <tabstop>leMemSize</tabstop>
  <tabstop>leDiskSize</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>leUserPassword</tabstop>
  <tabstop>leVerifyPassword</tabstop>
  <tabstop>leUserComment</tabstop>
  <tabstop>leUserHomeDir</tabstop>
  <tabstop>leUserShell</tabstop>
  <tabstop>leHttpProxy</tabstop>
  <tabstop>leHttpsProxy</tabstop>
  <tabstop>leFtpProxy</tabstop>
  <tabstop>leRsyncProxy</tabstop>
  <tabstop>leNoProxy</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VmSettings_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VmSettings_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
